recipes/funcsigs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - python-{{ name }} {{ version }}
+
+test:
+  imports:
+    - funcsigs
+
+about:
+  home: http://funcsigs.readthedocs.org
+  license: Apache 2.0
+  summary: Python function signatures from PEP362 for Python 2.6, 2.7 and 3.2+.
```

Comment:
Yep, I understand the onus.  I'll try to design for this.  I think the main thing is that we need access to metadata from any package listed in either the build or the run requirements - from there, it's a simple matter of putting in the dictionary access in the template.